oci/oci.go

```diff
@@ -67,6 +68,15 @@ func New(runtimeTrustedPath string,
 	logSizeMax int64,
 	noPivot bool,
 	ctrStopTimeout int64) (*Runtime, error) {
+	if runtimeTrustedPath == "" {
+		// this means no "runtime" key in config as it's deprecated, fallback to
+		// the runtime handler configured as default.
+		r, ok := runtimes[defaultRuntime]
+		if !ok {
+			return nil, fmt.Errorf("no runtime configured for default_runtime=%q", defaultRuntime)
```

Comment:
I feel like we should enforce a check that this default runtime is present from `validateRuntimeConfig()`.

Reply:
there's already such check and indeed this is redundant, but still, I want this here for further confirmation